orm/models.py

```diff
@@ -156,13 +156,22 @@ def select_related(self, related):
             select_related=related,
         )
 
+    async def exists(self) -> bool:
+        expr = self.build_select_expression()
+        expr = sqlalchemy.exists(expr).select()
+        exists, = await self.database.fetch_one(expr)
```

Comment:
What's being unpacked here?

Reply:
For some reason .fetch_one() returns a 1-tuple, eg (True,). I think using .scalar() may help here as you pointed in your other comment. :)